common-lib/common/panels/all/stat/info.libsonnet

```diff
@@ -0,0 +1,19 @@
+local g = import '../../../g.libsonnet';
+local base = import './base.libsonnet';
+local stat = g.panel.stat;
+//simple info panel with text or count
+base {
+  new(title, targets, description=''):
+    super.new(title, targets, description)
+    // Style choice: Make it bigger
```

Comment:
Can these style choices be overridden? Perhaps they could be encapsulated in an additional `stylize` function which adds them all, which could then also be used on dashboards/panels created with grafonnet directly?

Reply:
well, everything can be overriden, since it is jsonnet. We just providing defaults we think best.

Reply:
@rgeyer , good idea with stylize. Added in the latest commit.

Here is example how to transform panel created by other libs:
```
      idleCPU::
        nodePanels.timeseries.new(
          'CPU Usage',
          description='Total CPU utilisation percent.'
        )
        .withStacking('normal')
        .addTarget(c.commonPromTarget(
          expr=c.queries.cpuUsagePerCore,
          legendFormat='cpu {{cpu}}',
        ))
        + commonlib.all.timeSeries.base.stylize(),
```

Reply:
Implemented stylize().

All panels have `stylize()` method now that can be used to 'mutate' old panels or change style of existing panels.
You can also use it as `stylize(allLayers=false)` to ignore base styles and apply only top layer.  This can be handy when you want to cherry-pick different style sets from different panels to compose new combination.